themes/bootstrap3/templates/Recommend/ExpandFacets.phtml

```diff
@@ -6,7 +6,7 @@
 <?php if ($expandFacetSet): ?>
   <div class="sidegroup">
  <?php foreach ($expandFacetSet as $title => $cluster): ?>
-    <h4><?=$this->transEsc($cluster['label']) ?></h4>
+    <h2><?=$this->transEsc($cluster['label']) ?></h2>
```

Comment:
I changed this to `h2` for consistency.  I'm not sure how it's used in practice, but I imagine I would put a static heading above it and make all of these actually `h3`s.  But I guess that would be up to an implementer, and `h2` seems safer here unless we actually add a static heading.

Reply:
I think long ago this was used by default in the Author module, but now it's probably never used (but kept around just in case). :-)

Reply:
This is also the only one I wonder about. It seems like this would be rendered from another template with a static header, but as Demian said, it's probably not used. I'll keep an eye out.

Reply:
Definitely not used by default; it's an option that can be enabled.

Before:
![image](https://github.com/vufind-org/vufind/assets/309069/c99f9b60-17cb-4d43-abe3-739d221f0345)

After:
![image](https://github.com/vufind-org/vufind/assets/309069/0f5f5415-4a17-4eaf-b73e-5e7eb6fcb875)

...this one might benefit from a top-level heading and making the individual sections smaller, but since I doubt anyone uses it, I don't think it's worth bothering to change. We can revisit if anyone complaints.